charts/tempo-distributed/templates/distributor/service-distributor.yaml

```diff
@@ -49,11 +49,17 @@ spec:
       protocol: TCP
       targetPort: zipkin
     {{- end }}
-    {{- if or (.Values.traces.otlp.http) (.Values.traces.otlp.grpc) }}
-    - name: distributor-otlp
-      port: 55680
+    {{- if .Values.traces.otlp.http }}
```

Comment:
thanks for double-checking! I though about keeping the legacy port too - but ended up it might be overkill...
yes this makes sense - is kind of a braking change - although probably not many are using otlp at this point

If you prefer, I can add both of them

Reply:
> If you prefer, I can add both of them

Yeah, i like this. We can just name the old one `-legacy` which will clue people into why there's two and give them the ability to transition to the new. It also means we can merge now and not deal with any timing elements related to 0.7.0

Reply:
added the port for compability - and mapped it to the actual grpc port 4317 of the container
> https://github.com/open-telemetry/opentelemetry-specification/pull/1221/files

```
    {{- if .Values.traces.otlp.grpc }}
    - name: distributor-otlp-grpc
      port: 4317
      protocol: TCP
      targetPort: otlp-grpc
    - name: distributor-otlp-legacy
      port: 55680
      protocol: TCP
      targetPort: otlp-grpc
    {{- end }}
```